spec/requests/lobbies_spec.rb

```diff
@@ -226,4 +226,64 @@
       expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::PLAYER_JOIN, { id: Player.last.id })
     end
   end
-end
+
+  describe 'start' do
```

Comment:
You could also test that the `Lobby` gets updated with status `in_progress` and that the API responds with the updated model.